SMBPositionAnalytics.md

```diff
@@ -54,14 +52,20 @@ curl -X POST \
 Currently only Carotene v3.1 is supported. See the documentation for [Job Title service](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/JobTitle.md)
 for more information about carotene classification and carotene IDs.
 
-## Carotene Response Structure
+## Responses
+The data returned will contain an object or group of objects representing actions taken on a carotene title or job respectively. Each action will have associated `current` and `delta` values. The `current` value will account the total number an action was performed over the last day; the `delta` will identify the change in percentage of the `current` value from the day before. Positive values in the `delta` identify an increase day over day and negative values identify a decrease.
+
+ Each response will return a trailing`timestamp` string. This is the date time in which the data was last refreshed. The `timestamp` string is returned in ISO 8601 format with UTC offset(yyyy-MM-dd'T'HH:mm:ss-XXX). Clients can expect that data will be refreshed daily to account for the delta change in actions performed.
```

Comment:
Beef comment: strictly speaking, this is ISO 8601 with Timezone offset, but it's not UTC because UTC refers specifically to offset 0 (or 'Z' as the offset at the end).

Reply:
Well, I was wrong and "UTC offset" appears to actually be correctly used here (Wikipedia and the RFC say so). Please ignore me.